src/gluonts/model/deepar/_estimator.py

```diff
@@ -109,6 +106,9 @@ class DeepAREstimator(GluonEstimator):
     time_features
         Time features to use as inputs of the RNN (default: None, in which
         case these are automatically determined based on freq)
+    _num_eval_samples_per_ts
```

Comment:
Looks good overall, thanks!

Perhaps a different name like `num_parallel_samples` is clearer? If the leading `_` doesn't work we can comment that this is just an optimization that does not influence the accuracy.